src/components/Organizations/Detail.tsx

```diff
@@ -0,0 +1,132 @@
+import { useOrganizationElections } from '~queries/organizations'
+import { LoadingCards } from '~src/layout/Loading'
+import { PaginationProvider, usePagination } from '~components/Pagination/PaginationProvider'
+import { Pagination } from '~components/Pagination/Pagination'
+import ShowRawButton from '~src/layout/ShowRawButton'
+import { ElectionCard } from '~components/Process/Card'
+
+const OrganizationDetail = () => {
+  const { organization: org } = useOrganization()
+
+  // Should be already loaded
+  if (!org) return null
+
+  const id = org.address
+  const isSmallScreen = useBreakpointValue({ base: true, sm: false })
+
+  return (
+    <>
+      <OrganizationHeader fallbackSrc={FallbackHeaderImg} />
+      <Flex gap={4} direction={'column'} align={'center'} mt={{ base: '-80px', md: '-100px' }}>
+        <OrganizationImage
+          objectFit='cover'
+          maxW={{ base: '80px', md: '120px', lg: '140px' }}
+          maxH={'140px'}
+          fallbackSrc={FallbackAccountImg}
+        />
+        <VStack>
+          <OrganizationName fontSize='4xl' />
+          {isSmallScreen ? (
+            <ReducedTextAndCopy color={'textAccent1'} toCopy={id} fontWeight={'normal'} h={0} fontSize={'md'}>
+              {id}
+            </ReducedTextAndCopy>
+          ) : (
+            <CopyButton toCopy={id} color={'textAccent1'} fontWeight={'normal'} h={0} fontSize={'md'}>
+              {id}
+            </CopyButton>
+          )}
+          <Flex
+            as={'a'}
+            target='blank'
+            href={`${AppBaseURL}/organization/${ensure0x(id)}`}
+            pt={4}
+            align={'end'}
+            gap={3}
+            color={'blueText'}
+          >
+            <Box>
+              <Icon as={FaUserAlt} boxSize={5} />
+            </Box>
+            <Box>
+              <Text fontSize='xl' verticalAlign='bottom'>
+                <Trans i18nKey={'organization.view_profile'}>(View profile)</Trans>
+              </Text>
+            </Box>
+          </Flex>
+        </VStack>
+      </Flex>
+      <Flex align='start' gap={2}>
+        {org.account.description.default && (
+          <>
+            <Text fontSize='xl' color={'blueText'}>
+              <Trans i18nKey={'organization.description'}>Description</Trans>
+            </Text>
+            <OrganizationDescription />
+          </>
+        )}
+      </Flex>
+      <Text fontSize='xl' color={'blueText'}>
+        <Trans i18nKey={'organization.elections_list'}>Elections List:</Trans>
+      </Text>
+      <OrganizationElections org={org} />
+      <ShowRawButton obj={org} />
+    </>
+  )
+}
+
+const OrganizationElections = ({ org }: { org: AccountData }) => {
+  if (org.electionIndex === 0) {
+    return (
+      <Text>
+        <Trans i18nKey={'organization.no_elections'}>No elections yet!</Trans>
+      </Text>
+    )
+  }
+
+  return (
+    <PaginationProvider totalPages={Math.ceil(org.electionIndex / 10)}>
```

Comment:
Let's define a `paginationItemsPerPage` const and stop setting magic numbers in the code.

Reply:
I understand that what do you want is to instead of pass a `totalPages` attribute use something like `totalElements` and `itemsPerPage` and then do de calculations inside the pagination. 

If is this, I suggest to do it on the #12 , which refactors the paginator, so we can close and merge this

Reply:
Nah I was just suggesting to have that `10` into a constant, because we're using it in many places and, in a near future, we'll be able to define such limit on the API.